homeassistant/components/bmw_connected_drive/lock.py

```diff
@@ -45,17 +44,17 @@ async def async_setup_entry(
 class BMWLock(BMWBaseEntity, LockEntity):
     """Representation of a MyBMW vehicle lock."""
 
+    entity_description = LockEntityDescription(key="lock", name="Lock")
```

Comment:
Since this is hardcoded, maybe just do:
```suggestion
    _attr_name = "Lock"
```